src/engine/SCons/Tool/suncxx.py

```diff
@@ -74,7 +74,7 @@ def get_package_info(package_name, pkginfo, pkgchk):
         except EnvironmentError:
             pass
         else:
-            pkginfo_contents = p.communicate()[0]
+            pkginfo_contents = p.communicate()[0].decode()
```

Comment:
How about `universal_newlines=True` in the `subprocess` call instead?  I guess it's up to what the maintainer prefers...

Reply:
I guess both can be used here - I personally prefer `decode()` so I used it here, however as you said, whatever the maintainer likes the most.